CliClient/tests/models_Note.js

```diff
@@ -129,4 +134,98 @@ describe('models_Note', function() {
 		expect(duplicatedNote.user_updated_time !== note1.user_updated_time).toBe(true);
 	}));
 
+	it('should delete a set of notes', asyncTest(async () => {
+		let folder1 = await Folder.save({ title: 'folder1' });
+		let noOfNotes = 20;
+		for (let i = 0; i < noOfNotes; i++) {
+			await Note.save({ title: `note1${i}`, parent_id: folder1.id });
+		}
+
+		let noteIds = await Folder.noteIds(folder1.id);
+		await Note.batchDelete(noteIds);
+
+		const all = await allItems();
+		expect(all.length).toBe(1);
+		expect(all[0].id).toBe(folder1.id);
+	}));
+
+	it('should delete only the selected notes', asyncTest(async () => {
+		let f1 = await Folder.save({ title: 'folder1' });
+		let f2 = await Folder.save({ title: 'folder2', parent_id: f1.id });
+
+		let noOfNotes = 20;
+		for (let i = 0; i < noOfNotes; i++) {
+			await Note.save({ title: `note1${i}`, parent_id: f1.id });
+		}
+		for (let i = 0; i < noOfNotes; i++) {
+			await Note.save({ title: `note2${i}`, parent_id: f2.id });
+		}
```

Comment:
Instead use `createNTestNotes` from `test-utils.js`  (also applies to next test case)